packages/edit-site/src/components/page-pages/index.js

```diff
@@ -167,6 +191,7 @@ export default function PagePages() {
 				id: 'status',
 				getValue: ( { item } ) =>
 					postStatuses[ item.status ] ?? item.status,
+				...coreFieldTypes.string,
```

Comment:
So far, this is the only change of this PR, correct? `status` will now have a `render` (aka `cell`) while it didn't before. Presumably because tanstack handled it for us, and now we handle it directly via the core's types.